### DISCUSSION TITLE
option to pass "-lac" parameter to wiresock-client.exe

### DISCUSSION THREAD

**Discussion by naox** [wiresock/WireSockUI#57] [Ideas]
Please make option to pass "-lac" parameter to wiresock-client.exe

` -lac                     Spawn Virtual Network Interface associated with the tunnel (optional).`

Interface is usefull. You can bind app to such interface thus acting as fail safe.

**Comment by wiresock** [comment]
To access this feature, go to Settings and enable Virtual Adapter Mode.
